.gitignore

```diff
@@ -0,0 +1,10 @@
+
+*.bin
+data/DSMRsettings.ini
+data/RINGdays.csv
+data/RINGhours.csv
+data/RINGmonths.csv
+DSMRloggerAPI.ino.generic.bin
+DSMRloggerAPI.ino.generic.bin
+*.bin
+DSMRloggerAPI.ino.generic.bin
```

Comment:
Mag ik hieruit opmaken dat DSMRloggerAPI.ino.generic.bin niet in github moet komen?
Zou 1x niet genoeg zijn?